<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Claus</class>
 <widget class="QWidget" name="Claus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>39</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>530</width>
    <height>39</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Attribute name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Attribute value</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
